src/lwc/geTemplateBuilderFormFields/geTemplateBuilderFormFields.js

```diff
@@ -604,4 +644,21 @@ export default class geTemplateBuilderFormFields extends LightningElement {
     handleBodyReadMore() {
         this.isReadMoreActive = true;
     }
+
+    /*******************************************************************************
+     * @description Method generates the required fields text attached to the validation message.
+     *
+     * @param {object} objectData: Returned Object Schema
+     */
+    buildRequiredFieldsMessage(objectData){
+        for (let i= 0; i< REQUIRED_FORM_FIELDS.length; i++) {
+            let character;
+            if (i < REQUIRED_FORM_FIELDS.length - 1) {
+                character = COMMA_CHARACTER;
+            }else if ( i === REQUIRED_FORM_FIELDS.length - 1) {
+                character = PERIOD_CHARACTER;
+            }
+            REQUIRED_FORM_FIELDS_MESSAGE += objectData.fields[REQUIRED_FORM_FIELDS[i]].label + character;
```

Comment:
I'm still not sure what I'm doing to reproduce the issue, but I'm occasionally getting repeated fields in the error message (see screenshot). Do we need to clear this array out at the beginning of this method, since the variable is in scope and continually appended to? 

![image](https://user-images.githubusercontent.com/45107660/72194198-9b929100-33c1-11ea-8c39-4af9607ace6c.png)

Reply:
To handle it now, I'm clearing out the string variable just before I start building the message, I still haven't been able to reproduce it though

Reply:
I don't think I can reproduce it with those changes now either! Thanks!